domador.js

```diff
@@ -370,12 +371,17 @@ Domador.prototype.process = function process (el) {
     return;
   }
 
+  if (this.inTable) console.log("INTABLE ", el.tagName);
+  if (this.inTable && ['DIV', 'P', 'UL', 'LI', 'PRE', 'BLOCKQUOTE'].indexOf(el.tagName) !== -1) {
```

Comment:
Use all of `paragraphTags` plus `'P', 'UL', 'LI', 'PRE'` here. Maybe name it `blockTags` below `paragraphTags`.